Sources/MessagingPush/hooks/ModuleHook.swift

```diff
@@ -28,4 +28,8 @@ class MessagingPushModuleHookProvider: ModuleHookProvider {
     var deviceAttributesHook: DeviceAttributesHook? {
         MessagingPushImplementation(siteId: siteId)
     }
+
+    var screenTrackingHook: ScreenTrackingHook? {
```

Comment:
is this needed even though it returns nil?

Reply:
In order for the class to satisfy the requirements of the protocol, yes, this property is required in the code for it to compile. 

I could fix this by introducing a class that gets extended... 🤔